src/components/ui/ScheduleItem/ScheduleItem.tsx

```diff
@@ -0,0 +1,42 @@
+import classNames from "classnames";
+import styles from "./ScheduleItem.module.scss";
+import Icon from "@mdi/react";
```

Comment:
We already have `Icon` component in `src/components/ui/Icon/Icon.tsx`. Can we decide on one implementation (the one from `components/ui` or `@mdi/react`) and re-export used icon paths in one file (for example `src/components/ui/Icon/index.ts`) to keep track on which icons are used and do our best to reduce number of used icons in order to make the views as consistent as possible?

Reply:
Sorry, I am blind 🤦‍♂️ I forgot that we have special component for that - I will change it and remove `@mdi/react` from the project.